.pre-commit-config.yaml

```diff
@@ -39,3 +39,8 @@ repos:
     hooks:
     -   id: mypy
         exclude: ^(docs/|example-plugin/)
+-   repo: https://github.com/codespell-project/codespell
+    # Configuration for codespell is in setup.cfg
+    rev: v2.2.6
+    hooks:
+    -   id: codespell
```

Comment:
note -- this is just a check, not a fixer, so if false positive - just add a skip in setup.cfg; i.e. there would be no bugs introduced by codespell